### PR TITLE
Fix OutputWalker conflict #181

### PR DESCRIPTION
**Pull request by aminin**

### CODE REVIEW

File: src/Knp/Component/Pager/Event/Subscriber/Paginate/Doctrine/ORM/QuerySubscriber.php [hunk 1]
```diff
@@ -33,6 +33,7 @@ public function items(ItemsEvent $event)
             if (version_compare(\Doctrine\ORM\Version::VERSION, '2.3.0', '>=')) {
                 $useDoctrineWalkers = true;
                 $useDoctrineOutputWalker = true;
+                $useDoctrineOutputWalker = ($event->target->getHint(Query::HINT_CUSTOM_OUTPUT_WALKER) === false);
```
Comment (roukmoute) [inline src/Knp/Component/Pager/Event/Subscriber/Paginate/Doctrine/ORM/QuerySubscriber.php]: Remove line 35.

Comment (aminin) [reply src/Knp/Component/Pager/Event/Subscriber/Paginate/Doctrine/ORM/QuerySubscriber.php]: done

**Comment by roukmoute** [comment]
There is no tests ?

**Comment by aminin** [comment]
@roukmoute, I can't reproduce the case in test env :unamused:

**Comment by garak** [comment]
Just add a reference to issue #181 (since it's not linked when in title)

**Comment by garak** [comment]
@aminin is this PR relevant for latest branch too?

**Comment by garak** [comment]
Closing for missing feedback

**Pull request closed without merging**